lisa/tools/aria.py

```diff
@@ -0,0 +1,83 @@
+from pathlib import PurePosixPath
+from typing import TYPE_CHECKING
+
+from lisa.base_tools import Wget
+from lisa.executable import Tool
+from lisa.tools.make import Make
+from lisa.tools.tar import Tar
+
+if TYPE_CHECKING:
+    from lisa.operating_system import Posix
```

Comment:
the os reference won't bring cycle import for this tool. If so, just import as normal.

Reply:
The bzip2 is not changed.